test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/DynamicGasCostSuite.java

```diff
@@ -953,9 +954,8 @@ private HapiApiSpec priorityAddressIsCreate2ForStaticHapiCalls() {
 									staticCallAliasAns.get(),
 									staticCallMirrorAns.get(),
 									"Static call with mirror address should be same as call with alias");
-							assertEquals(
-									staticCallAliasAns.get().toString(16),
-									aliasAddr.get(),
+							assertTrue(
+									aliasAddr.get().endsWith(staticCallAliasAns.get().toString(16)),
```

Comment:
This is not good! The expected string should be padded with zeros to a certain number of digits (40, I believe). Otherwise, there is a chance for this test to pass unexpectedly!

I am surprised that there is no `BigInteger.toString(radix, digits)`. The implementation of `BigInteger.toString(radix)` that we are using makes a call to `toString(abs, sb, radix, 0)` with 0 as the digits meaning no padding, and unluckily this function is `private`.

Reply:
We could instead do,
```
assertEquals(new BigInteger(aliasAddr.get(), 16), staticCallAliasAns.get(), ...)
```
...but I think the probability of a **genuine** false positive is ~1 in 2^60, so we're likely safe either way.

Reply:
>  and unluckily this function is private

it is private Quan. Will convert alias to BigInt and do the comparison instead.